Alert/alert.js

```diff
@@ -0,0 +1,8 @@
+const closeButton = document.querySelectorAll('.alert');
+closeButton.forEach(btn => btn.addEventListener('click', close));
```

Comment:
Nice that you have taken in account that a single page can have multiple references.